pci-compliance/saqs.md

```diff
@@ -8,28 +8,22 @@ To be [PCI Compliant](/pci-compliance/) as a merchant, you'll need to fill out a
 
 Most likely, the 2 SAQ's that are relevant to you if you accept payments online are [SAQ A](#saq-a) and [SAQ A-EP](#saq-a-ep). For a full list, see [Understanding the SAQs for PCI DSS version 3](https://www.pcisecuritystandards.org/documents/Understanding_SAQs_PCI_DSS_v3.pdf).
 
-> Following SAQ rules is not enough to ensure a secure payments flow (todo: maybe link to a security section?).
+Following SAQ rules is not enough to ensure a secure payments flow. Keeping up with industry-standards is important to make sure payment information stays secure.
 
-# SAQ A
+## SAQ A
 
 The easiest. Requires use of an iframe or a redirect so that payment information is entered directly on the [provider's](/glossary/#provider) domain.
 
 iOS and Android payment integrations still qualify for SAQ A without needing to employ iframes or redirects.
```

Comment:
That's my understanding. There might be other reasons as well, I'm not super familiar with PCI rules wrt mobile.